boot/bootutil/include/bootutil/bootutil_public.h

```diff
@@ -71,7 +71,17 @@ extern "C" {
 /** Swapping encountered an unrecoverable error */
 #define BOOT_SWAP_TYPE_PANIC    0xff
 
+#define BOOT_MAGIC_SZ           16
+
+#ifdef MCUBOOT_BOOT_MAX_ALIGN
+#define BOOT_MAX_ALIGN          MCUBOOT_BOOT_MAX_ALIGN
+#define BOOT_MAGIC_ALIGN_SIZE \
+    ((((BOOT_MAGIC_SZ - 1) / BOOT_MAX_ALIGN) + 1) * BOOT_MAX_ALIGN)
```

Comment:
This expression for aligning may be extracted to a generic macro to improve code comprehension. See:
https://github.com/mcu-tools/mcuboot/blob/main/boot/espressif/port/esp_mcuboot.c#L28
https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/const.h#L31